bbb-graphql-server/metadata/databases/BigBlueButton/tables/public_v_audio_caption.yaml

```diff
@@ -0,0 +1,29 @@
+table:
+  name: v_audio_caption
+  schema: public
+configuration:
+  column_config: {}
+  custom_column_names: {}
+  custom_name: audio_caption
+  custom_root_fields: {}
+object_relationships:
+  - name: audio_caption_user_ref
```

Comment:
```suggestion
  - name: user
```

Reply:
The name of the relation needs to be `user`, then the query to get the user name will be like:

```
subscription {
  audio_caption {
    transcriptId
    transcript
    user {
      name
    }
  }
}

```